### DISCUSSION TITLE
![image] tag causes failure to build on Github Pages with "error calling Resize: must provide Width or Height"

### DISCUSSION THREAD

**Discussion by EndianTribe** [thuliteio/doks#932] [Q&A] [answered]
I have searched for similar issues on Google and here on Github to no avail. I hope members of the community can shed some light on this.

## Issue description
`![image]` tag causes failure to build on Github Pages with `error calling Resize: must provide Width or Height`

## Directory structure for images
![Screen Shot 2022-10-28 at 01 22 51](https://user-images.githubusercontent.com/45805291/198541119-38b02ffc-ab70-4ba2-bdcb-c93a5bcf2774.png)

I believe I followed the instructions on page bundle.

## Shortcode used
![Screen Shot 2022-10-28 at 01 23 11](https://user-images.githubusercontent.com/45805291/198541193-9809a60a-533c-408c-bbc4-cb3a1edb4864.png)

## Build platform

Github Actions

## Full error message

```
Error: Error building site: "/home/runner/work/Reconcile-Website/Reconcile-Website/content/en/docs/under_the_hood/internal-testing/index.md:1:1": "/home/runner/work/Reconcile-Website/Reconcile-Website/node_modules/@hyas/images/layouts/_default/_markup/render-image.html:8:3": execute of template failed: template: _default/_markup/render-image.html:8:3: executing "_default/_markup/render-image.html" at <partial "lazyimg" $params>: error calling partial: "/home/runner/work/Reconcile-Website/Reconcile-Website/node_modules/@hyas/images/layouts/partials/lazyimg.html:58:21": execute of template failed: template: partials/lazyimg.html:58:21: executing "partials/lazyimg.html" at <partial (printf "resizer/%s" $params.resizer) $params>: error calling partial: "/home/runner/work/Reconcile-Website/Reconcile-Website/node_modules/@hyas/images/layouts/partials/resizer/responsive.html:7:17": execute of template failed: template: partials/resizer/responsive.html:7:17: executing "partials/resizer/responsive.html" at <partial "resizer/simple" .>: error calling partial: "/home/runner/work/Reconcile-Website/Reconcile-Website/node_modules/@hyas/images/layouts/partials/resizer/simple.html:11:16": execute of template failed: template: partials/resizer/simple.html:11:16: executing "partials/resizer/simple.html" at <$res.Resize>: error calling Resize: must provide Width or Height
```

## Link to repo and problem page
https://github.com/EndianTribe/Reconcile-Website/blob/master/content/en/docs/under_the_hood/internal-testing/index.md

Thanks in advance for your assistance!

**Comment by h-enk** [comment, marked as answer]
Does this help you: https://github.com/h-enk/doks/discussions/922#discussioncomment-3857518 ?

**Reply by EndianTribe** [reply]
Hi h-enk,

Thanks for your quick response! The two sections indicated in the linked comment were already in the `params.toml` file as you can see below.

![Screen Shot 2022-10-28 at 01 59 24](https://user-images.githubusercontent.com/45805291/198548360-a3a631df-ef2d-4de4-98a8-406b10c251e8.png)

**Reply by h-enk** [reply]
The Doks child theme [^1] does not support v0.3.0 of @hyas/images. Please revert back to 0.2.3:

```bash
npm i @hyas/images@0.2.3
```

Also, in `./config/_default/module.toml`, make sure to have this block:

```toml
[[mounts]]
  source = "node_modules/@hyas/images/layouts"
  target = "layouts"
```

**above** this block:

```toml
[[mounts]]
  source = "layouts"
  target = "layouts"
```

And (unrelated): rename `./content/en/contact/_index.md` to `./content/en/contact/index.md`

[^1]: I will be soon releasing the new restructured Hyas — it's the new base for Doks (making the Doks child theme obsolete).

**Reply by EndianTribe** [reply]
Hi h-enk,

Thanks much for the update. This indeed fixed my issue. i added the install instruction to two of the three github action files and they worked fine.

Thanks again for your help!
